samples/nrf9160/aws_fota/Kconfig

```diff
@@ -44,6 +44,12 @@ config CLOUD_CLIENT_ID
 	default "your_client_id"
 endif
 
+if !USE_CLOUD_CLIENT_ID
```

Comment:
please change into a `depends on !USE_CLOUD_CLIENT_ID` on the specific config.

Reply:
Ah, I see. I was not sure what approach is preferred.

Will change.